docs/_static/theme_overrides.css

```diff
@@ -1,7 +1,38 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
```

Comment:
This makes our docs online-only. They are intended to be distribute-able offline. What is the licens of this icon stuff, are we permitted to copy it into our own source?

Reply:
Yes, for now it's online only, same as our docs, but a set of icons can easily be included once we start exploring the option of making offline docs.

License is not the issue, as these are exactly the same icons we use in Kolibri.

Reply:
Alright, cool, then let's open an issue to put this directly in the docs as static assets :)

Reply:
Yes, not necessary to include in this release (we might need to make a subset of MD icons, no need to include all), but as soon as offline docs are a reality, the issue to remind us will be handy).